### ISSUE TITLE
Delay in acquisition

### ISSUE THREAD

**Issue by bysjsys** [basler/pypylon#769]
### Describe what you want to implement and what the issue & the steps to reproduce it are:

My project is used for industrial vision detection, and the trigger wait time needs to be increased to a larger value. But this leads to a longer time (about 0.1s) between the camera trigger and receiving the picture. It seems not significant but did affect the detection efficiency. I want to improve the trigger wait time while reducing the delay to maintain a fast acquisition speed. I tried setting TriggerDelay but it didn't seem to work

### Is your camera operational in Basler pylon viewer on your platform

Yes

### Hardware setup & camera model(s) used

Windows 10, X86_64, acA2440-75uc

### Runtime information:

```shell
python: 3.9
pypylon: 7.1.0
```

**Comment by thiesmoeller** [comment]
Do you use a hardware trigger?
What do you mean by trigger wait time?

**Comment by bysjsys** [comment]
Yes, we use a hardware trigger. Trigger wait time is originally set as 5000ms in RetrieveResult(). I means there is a delay time between the camera is triggered and the graph is acquired in software. We found when the trigger wait time is set as a larger value, such as 1000000ms, this delay time will be incresed. Even if this delay time is short (0.1s) but still can not meet our needs with high efficiency.

**Comment by thiesmoeller** [comment]
The value you give for RetrieveResult is only the maximum time to wait for an image to arrive. It has no impact on the latency from trigger to image.

The time you perceive is the time the camera needs to expose readout and transfer the image data.

What is your camera model, ROI, Pixelformat and exposure time ?

**Comment by SMA2016a** [comment]
I think using Onimage Callback might be  also a good solution.
